py/sentry_relay/processing.py

```diff
@@ -261,6 +262,22 @@ def validate_project_config(config, strict: bool):
         raise ValueError(error)
 
 
+def normalize_global_config(config):
+    """Normalize the global config.
+
+    Normalization consists of deserializing and serializing back the given
+    global config. If deserializing fails, throw an exception. Note that even if
+    the roundtrip doesn't produce errors, the given config may differ from
+    normalized one.
+
+    :param config: the global config to validate.
+    """
+    serialized = json.dumps(config)
+    normalized = rustcall(lib.normalize_global_config, encode_str(serialized))
+    decoded = decode_str(normalized, free=True)
+    return json.loads(decoded)
```

Comment:
Sorry for going in circles here, but I think this would provide the most value (in combination with other suggestions): 

```suggestion
    try:
        return json.loads(decoded)
    except json.JSONDecodeError:
        raise ValueError(decoded)
```